hushline/routes.py

```diff
@@ -344,12 +352,15 @@ def verify_2fa_login() -> Response | str | tuple[Response | str, int]:
                 rate_limit = True
 
             # If there were 5 failed logins in the last 30 seconds, don't allow another one
-            failed_logins = (
-                AuthenticationLog.query.filter_by(user_id=user.id, successful=False)
-                .filter(AuthenticationLog.timestamp > datetime.now() - timedelta(seconds=30))
-                .count()
+            failed_logins = db.session.scalar(
+                db.select(
+                    db.func.count(AuthenticationLog.id)
+                    .filter(AuthenticationLog.user_id == user.id)
+                    .filter(AuthenticationLog.successful == db.false())
```

Comment:
this one gave me a bit of trouble because if you use `False` instead of `db.false()` the linter is unhappy and recommends something that isn't equivalent.